files/zh-cn/web/css/text-size-adjust/index.md

```diff
@@ -7,6 +7,14 @@ slug: Web/CSS/text-size-adjust
 
 **`text-size-adjust`** [CSS](/zh-CN/docs/Web/API/CSS) 控制将一些手机或平板设备上使用的文本溢出算法（text inflation algorithm）。其他类型的设备上的浏览器会忽略此属性。
 
+因为许多网站还没有适配屏幕较小的设备，移动设备的浏览器和桌面浏览器在渲染网页时可能会有不同。他们不是以设备屏幕宽度布局网页，而是用比屏幕宽一些的 {{glossary("viewport", "视窗")}} 去布局网页，通常是 800 到 1000 像素。为了将视窗上的布局映射到原始设备屏幕上，手机浏览器要么只渲染整个页面的一部分，要么将视窗缩放至原始屏幕大小。
+
+因为缩放适配小屏幕而导致文字会变得很小，许多手机浏览器会使用文本溢出算法放大文本，改善可读性。当一个包含文本的元素使用了 100% 的屏幕宽度，该算法会增加文本大小，但是不会修改布局。`text-size-adjust` 这个属性允许开发者去除或者修改浏览器的这种行为，比如，当网页已经适配了小屏幕之后，就不需要这么做了。
+
+{{cssinfo}}
+
```

Comment:
The modification here seems to be a mistake. Please check  [the upstream](https://github.com/mdn/content/blob/3f4eae0b1389c7f565cfc000e9196908da003b82/files/en-us/web/css/text-size-adjust/index.md?plain=1#L47-L53)

Reply:
It seems like just adding a `Formal definition` header is required ?

Reply:
No. This part is after the "Syntax" section. Could you see it

Reply:
Oh, I just saw that `Formal syntax` was confused, thank you.